docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +296,46 @@ To enable denormals optimization in the application, the `denormals_optimization
 
 @endsphinxdirective
 
+### Sparse weights decompression
+There is a known issue that the computing power of modern hardware is growing faster than memory. Using more and more modern HW CPUs, we may never reach the theoretical peak performance of some models due to the memory bound problem. `Sparse weights decompression feature` allows to achieve greater performance on such models just due to more reasonable memory consumption.
```

Comment:
"computing power of modern hardware is growing faster than memory" - we cannot do such statements. It is not fully true even for all Intel x86 HW, not talking about other architectures. 

I would propose to rearrange the doc with following.

- First segment introduces the meaning of sparse weights (not matrix) and sparse rate.
- Second segment states that we can utilize sparse weights in order to achieve better performance for workloads which are highly memory bound (e.g. throughput scenario) using `Sparse weights decompression` feature + how it works.
- Third segment describes how to use the feature from API perspective + code samples. It also worth to add example how to chech that the layer was really executed with sparse feature enabled (copy of perf counters log)
- Lastly you can add limitation section there you descibe that the feature works only for Matrix Multiplication operation (not FullyConnected - this is internal op) and supported only on HW targets with Intel AMX extension suppport (e.g. SPR)
- Also as Maxim mentioned you should explain that feature is disabled by default since it overall speedup highly depends on particular workload and for some cases the feature may intoduce performance degradations. 
-

Reply:
changed, thanks!